### DISCUSSION TITLE
facebook integration ( authorization error)

### DISCUSSION THREAD

**Discussion by scaladad** [chatwoot/chatwoot#7564] [General]
I use Facebook api version 17 for self-hosted Chatwoot as it is only one available ( there is no option to switch to 14 version in Facebook settings).  It looks like my integration doesn't  work.  Any ideas?

**Comment by jtayrone** [comment]
Checks what versioning is in your version. You can do this by going to the file `/home/chatroot/chatroot/app/controllers/dashboard_controller.rb`, confirm that in `FACEBOOK_API_VERSION:` it is in version `v17.0`.

If your version is lower than version `17.0` change and recompile your chatwoot.

![fb](https://github.com/chatwoot/chatwoot/assets/34479816/1473a6d2-97e6-45f9-b110-b85cb5a2b202)

**Reply by scaladad** [reply]
Thanks!
